auth-web/src/models/statement.ts

```diff
@@ -17,3 +17,17 @@ export interface StatementListResponse {
   page: number
   total: number
 }
+
+export interface StatementRecipient {
+  authUserId: number
+  email: string
+  firstname: string
+  lastname: string
+  name?: string
```

Comment:
whats the name here?

Reply:
for displaying in the auto complete search, this will be removed while sending to backend. thats why kept as optional